shopify_webhook/tests/test_webhook.py

```diff
@@ -16,11 +16,11 @@ def test_empty_post_message_is_bad_request(self):
 
     def test_no_hmac_is_forbidden(self):
         response = self.post_shopify_webhook(topic = 'orders/create', data = {'id': 123}, send_hmac = False)
-        self.assertEqual(response.status_code, 403, 'POST orders/create request with no HMAC returns 403 (Forbidden).')
+        self.assertEqual(response.status_code, 401, 'POST orders/create request with no HMAC returns 403 (Forbidden).')
```

Comment:
Please update the assertion message too :)

Reply:
resolved in https://github.com/discolabs/django-shopify-webhook/commit/4312920ec138ec30f8523f637ba2e5df4b9de8bb